Taggers/python/tagsDumpConfig_cff.py

```diff
@@ -6,7 +6,8 @@
     generatorInfo = cms.InputTag("generator"),
     processId = cms.string(""),
     maxCandPerEvent = cms.int32(-1),
-    lumiWeight = cms.double(1.0),
+    lumiWeight = cms.double(1.0),# Over-written by metaData if using fggRunJobs
+    intLumi = cms.untracked.double(1000.), # in /pb. to be over-written by metaData if using fggRunJobs
```

Comment:
Sorry, I got confused about the IntLumi parameter being tracked or untracked. Please disregard this comment: the code is fine as it is.